strings/ion_laws.json

```diff
@@ -1,320 +1,345 @@
+        "CHAMSKI",
+        "NIEPRZYZWOITY",
+        "PUSTY",
+        "ZALANY",
+        "ELEKTRYCZNY",
+        "KRĘCĄCY SIĘ",
+        "MIĘSNY",
+        "NIEPRZYJAZNY",
+        "NIELEGALNY",
+		"KREMDKOWY",
+        "ROBOTYCZNY",
+        "MECHANICZNY",
+        "ORGANICZNY",
+        "ETERYCZNY",
+		"NIETRZEŹWY",
+        "PRZEZROCZYSTY",
+		"NIEOPISYWALNY",
+        "ODWRÓCONY",
+        "ŚWIECĄCY SIĘ",
+        "TRZĘSĄCY SIĘ",
+        "PIERDZĄCY",
+        "SRAJĄCY",
+        "ODBYJAJĄCY SIĘ",
+        "ODDANY",
+        "ZAMASKOWANY",
+        "NIEZIDENTYFIKOWANY",
+        "DZIWNY",
+        "GOŁY",
+        "ŁYSY",
+		"OWŁOSIONY",
+        "TWERKUJĄCY",
+        "ROZPIESZCZAJĄCY",
         "REDACTED",
-        "TACTICAL",
-        "TACTICOOL",
-        "RED",
-        "ORANGE",
-        "YELLOW",
-        "GREEN",
-        "BLUE",
-        "PURPLE",
-        "BLACK",
-        "WHITE",
-        "BROWN",
-        "GREY",
-        "RAINBOW",
-        "IMPERFECT",
-        "DERANGED"
+        "TAKTYCZNY",
+        "UBER TAKTYCZNY",
+        "CZERWONY",
+        "POMARAŃCZOWY",
+        "ŻÓŁTY",
+        "ZIELONY",
+        "NIEBIESKI",
+        "FIOLETOWY",
+        "CZARNY",
+        "BIAŁY",
+        "BRĄZOWY",
+        "SZARY",
+        "TĘCZOWY",
+        "CHIŃSKI",
+        "STARODAWNY"
     ],
     "ionallergy": [
-        "COTTON",
-        "CLOTHES",
-        "ACID",
-        "OXYGEN",
-        "CARBON DIOXIDE",
-        "ELECTRICITY",
-        "HUMAN CONTACT",
-        "CYBORG CONTACT",
-        "MEDICINE",
-        "FLOORS",
-        "PLASMA",
-        "SPACE",
-        "AIR",
-        "PLANTS",
-        "IRON",
-        "GLASS",
-        "BOOKS",
-        "ROBOTS",
-        "LIGHT",
-        "DARKNESS",
-        "PAIN",
-        "HAPPINESS",
-        "DRINKS",
-        "FOOD",
-        "CLOWNS",
-        "LIZARDS",
+        "WATA",
+        "UBRANIA",
+        "KWAS",
+        "TLEN",
+        "DWUTLENEK WĘGLA",
+        "EKETRYCZNOŚĆ",
+        "KONAKTY MIĘDZYLUDZKIE",
+        "KONTAKTY Z CYBORGAMI",
+        "LEKI",
+        "PODŁOGI",
+        "PLAZMA",
+        "KOSMOS",
+        "POWIETRZE",
+        "ROŚLINY",
+        "ŻELAZO",
+        "SZKŁO",
+        "KSIĄŻKI",
+        "ROBOTY",
+        "ŚWIATŁO",
+        "CIEMNOŚĆ",
+        "BÓL",
+        "RADOŚĆ",
+        "NAPOJE",
+        "JEDZENIE",
+        "KLAUNY",
+        "JASZCZURY",
         "HUMOR",
-        "WATER",
-        "SHUTTLES",
-        "NUTS",
-        "SUNLIGHT",
-        "SEXUAL ACTIONS",
-        "BLOOD",
-        "HEAT",
-        "COLD",
-        "EVERYTHING"
+        "WODA",
+        "PROMY KOSMICZNE",
+        "ORZECHY",
+        "ŚWIATŁO SŁONECZNE",
+        "AKTY SEKSUALNE",
+        "KREW",
+        "CIEPŁO",
+        "ZIMNO",
+        "WSZYSTKO"
     ],
     "ionallergysev": [
-        "DEATHLY",
-        "MILDLY",
-        "SEVERELY",
-        "CONTAGIOUSLY",
-        "NOT VERY",
-        "EXTREMELY"
+        "ZABÓJCZE",
```

Comment:
zabójcze => zabójczo, żeby był przysłówek
not very => nie bardzo raczej niż "nie wszystkie"